conf/wdio-browserstack.conf.js

```diff
@@ -0,0 +1,33 @@
+var defaults = require("./wdio.conf.js");
+var _ = require("lodash");
+
+var overrides = {
+  browserstackLocal: false,
+  user: process.env.BROWSERSTACK_USERNAME || 'BROWSERSTACK_USERNAME',
+  key: process.env.BROWSERSTACK_ACCESS_KEY || 'BROWSERSTACK_ACCESS_KEY',
+  specs: [
+    './test/specs/plain/*.js'
+  ],
+  capabilities: [{
+    maxInstances: 1,
+    'browserstack.debug': true,
+    'browserstack.video': true,
+    os: "OS X",
+    os_version: "Catalina",
+    browserName: 'Chrome',
+    browser_version: "latest",
+    acceptInsecureCerts: true,
+    name: 'BStack-Test',
+    build: 'BStack Build webdriverio single'
```

Comment:
Build name should be according this:
```
IF $BROWSERSTACK_BUILD_NAME is not empty or null
"<Repository name>" + $BROWSERSTACK_BUILD_NAME e.g. "browserstack-examples-cucumber-junit5 - " + $BROWSERSTACK_BUILD_NAME
ELSE
"<Repository name>" + <Date TS> e.g. "browserstack-examples-cucumber-junit5 - " + <Date TS>
```
Session name:
This can be set to the spec name is user is not passing any explicit session name via the command line.